magma/bits.py

```diff
@@ -541,6 +543,15 @@ def __getitem__(self, index):
             return (self >> index)[0]
         return super().__getitem__(index)
 
+    def orR(self):
```

Comment:
unless these names are standard or mirror the names in some other system (say, Chisel), I think doing `<op>_reduce` or `reduce_<op>` looks better :)

Reply:
Ok, I see the corresponding CoreIR primitives are `orr`, `xorr`, `andr`. I still think the function name can be the full word since this is a user-exposed function, and the CoreIR is mostly backend.

Reply:
Chisel uses orR, xoR, ... (that's where I got them), we could provide both.  One one hand, orR is significantly easier to type, on the other hand, maybe we shouldn't sacrifice ease of typing for readability (since autocomplete is prevalent these days), but I don't think orR, xorR, ... is too ambiguous to read easily.

Reply:
I think having both would be the least preferred option.

Personally prefer being on the side of the trade-off which is more clear and readable (also follows "code is the documentation" principle). But fine having `orR` etc. especially if it's in Chisel. Your call.